packages.json

```diff
@@ -3165,6 +3164,7 @@
   "https://github.com/webfrogs/transformers.git",
   "https://github.com/Weebly/OrderedSet.git",
   "https://github.com/weichsel/ZIPFoundation.git",
+  "https://github.com/weichsel/ZIPFoundation.git/",
```

Comment:
This one needs fixing, the others are all good changes.

Reply:
Fixed here: https://github.com/SwiftPackageIndex/Validator/commit/c9c2010b22f17c3ac9007915423f9934c0484c33

I'll trigger a re-run.

Reply:
Ok, this is now sorted and this PR is good to be merged.